bridgetown-website/src/_docs/plugins/html-inspectors.md

```diff
@@ -0,0 +1,65 @@
+---
+title: HTML Inspectors
+order: 0
+top_section: Configuration
+category: plugins
+---
+
+The HTML Inspectors API, added in Bridgetown 1.1, provides a useful and safe way to manipulate the HTML output of your resources. Safe because instead of using string manipulation, regular expressions, and the like—which is prone to error—you'll be working on real node trees. This is thanks to [Nokogiri](https://nokogiri.org), a Ruby gem which lets you work with a DOM-like API directly on HTML documents.
```

Comment:
> Safe because instead of ...

I think the beginning of this sentence needs a little love. What is safe? Why would it not be? etc.